pkg/proxy/proxy.go

```diff
@@ -165,6 +172,16 @@ func proxyTypeNotFoundError(proxyType types.ProxyType, listener string, ingress
 	return fmt.Errorf("unrecognized %s proxy type for %s: %s", dir, listener, proxyType)
 }
 
+func (p *Proxy) UpdateSDP(rules map[identity.NumericIdentity]policy.SelectorPolicy) {
+	if p.policyUpdater == nil {
```

Comment:
as far as i see the policyupdater should always be provided and never be `nil` - even if standalone dns proxy is disabled (it just doesn't start the server)